OracleWebLogic/samples/12213-webtier-apache/README.md

```diff
@@ -52,39 +61,69 @@ NOTE: To set the `DOMAIN_NAME`, you must set both `DOMAIN_NAME` and `DOMAIN_HOME
                      -e DOMAIN_HOME=/u01/oracle/user_projects/domains/abc_domain \
                      -e DOMAIN_NAME=abc_domain \
                      -p 7001:7001 \
-                     store/oracle/weblogic:12.2.1.3
+                     store/oracle/weblogic:<version>
```

Comment:
container-registry.oracle.com/middleware/weblogic:<version>

Reply:
Updated the README.md as per your comments. Please confirm if the directory name "12213-webtier-apache" need to be changed now as now the Dockerfile supports both 12.2.1.3 and 12.2.1.4.